R/ResamplingSpCVBuffer.R

```diff
@@ -72,47 +89,55 @@ ResamplingSpCVBuffer = R6Class("ResamplingSpCVBuffer",
     #'   Returns the number of resampling iterations, depending on the
     #'   values stored in the `param_set`.
     iters = function() {
-      as.integer(length(self$instance$fold))
+      as.integer(length(self$instance))
     }
   ),
 
   private = list(
-    .sample = function(ids, coords, crs) {
+    .sample = function(ids, response, coords, positive, crs, properties) {
 
       require_namespaces(c("blockCV", "sf"))
 
-      points = sf::st_as_sf(coords,
+      pars = self$param_set$get_values()
+
+      if (!isTRUE("twoclass" %in% properties) && isTRUE(pars$spDataType == "PB")) {
```

Comment:
Please remove `isTRUE()` calls.

Reply:
We can remove the first `isTRUE` but the second condition produces an error without `isTRUE` if `pars$spDataType` is not set (We would use the package default in this case). If you want we can set `pars$spDataType == "PA"` before checking the condition if `pars$spDataType` is not set.

Reply:
Hm I see. Maybe we should just leave it and `isTRUE` is not as bad as I think about it 😄